gslib/addlhelp/encoding.py

```diff
@@ -49,10 +49,7 @@
   If you encounter such an error you can either remove the problematic file(s)
   or try to rename them and re-run the command. If you have a modest number of
   such files the simplest thing to do is to think of a different name for the
-  file and manually rename the file (using local filesystem tools). If you have
-  too many files for that to be practical you can use a tool to convert the old
-  character encoding to UTF-8. One such tool is `native2ascii
-  <http://docs.oracle.com/javase/7/docs/technotes/tools/solaris/native2ascii.html>`_.
+  file and manually rename the file (using local filesystem tools). If you have.
```

Comment:
This line ends abruptly.

Reply:
Fixed! Looks like the paste got cut off.